### ISSUE TITLE
libpointmatcher fails to build on jenkins.ros.org, because libnabo has not been released since ...

### ISSUE THREAD

**Issue by HannesSommer** [ethz-asl/libnabo-release#1]
ethz-asl/libnabo@62a384d99e943c362b4895356119bdb9281caca3. 
http://jenkins.ros.org/job/devel-indigo-libpointmatcher/ looks so red because of the broken 

```
set(libnabo_LIBRARIES /tmp/buildd/ros-indigo-libnabo-1.0.5-0trusty-20141229-2203/obj-x86_64-linux-gnu/libnabo.a)
```

in /opt/ros/indigo/share/libnabo/cmake/libnaboConfig.cmake as part of `ros-indigo-libnabo` in version 
1.0.5-0trusty-20141229-2203-+00 (newest). This should be better with the mentioned commit.

**Comment by simonlynen** [comment]
I can push a new release tomorrow.

**Comment by HannesSommer** [comment]
That would be great. I you don't have the time I can also try to do that - but no experience, yet.

**Comment by simonlynen** [comment]
@HannesSommer that's a very nice offer. If you do have time and want to play around it is pretty easy and should only take ~15 minutes the first time you do it. You can basically just follow the guide from ROS for bloom to release packages. It was pretty straight forward the last time I tried. Let me know if you want to tackle this, otherwise I can do it later.

**Comment by HannesSommer** [comment]
Thanks for the hints. I'll give it a try.

**Comment by HannesSommer** [comment]
This is most likely fixed with 16b3c9d2f. 

Before there was 

```
[ 37%] Building CXX object CMakeFiles/pointmatcher.dir/pointmatcher/TransformationCheckersImpl.cpp.o
[ 39%] Building CXX object CMakeFiles/pointmatcher.dir/pointmatcher/InspectorsImpl.cpp.o
make[2]: *** No rule to make target `/tmp/buildd/ros-indigo-libnabo-1.0.5-0trusty-20141229-2203/obj-x86_64-linux-gnu/libnabo.a', needed by `libpointmatcher.so.1.2.1'.  Stop.
```

and now there is

```
[ 37%] Building CXX object CMakeFiles/pointmatcher.dir/pointmatcher/TransformationCheckersImpl.cpp.o
[ 39%] Building CXX object CMakeFiles/pointmatcher.dir/pointmatcher/InspectorsImpl.cpp.o
make[2]: *** No rule to make target `contrib/yaml-cpp-pm/libyaml-cpp-pm.a', needed by `libpointmatcher.so.1.2.1'.  Stop.
```

:+1: 

So libpointmatcher fails again, this time for libpointmatcher issues that were invisible before.
1. the latest release does fail now because it won't link to gomp.
2. the devel (current master) also fails but because it doesn't build `libyaml-cpp-pm.a` (see above)

I believe 1. is fixed already in master (the libraries of libnabo weren't properly propagated before). So a new release of libpointmatcher would solve it. 

But the second needs to be solved first. I'm pretty sure (based on comparison of jenkins.ros.org logfiles) that it was introduced with ethz-asl/libpointmatcher@a9b72eb323, (https://github.com/ethz-asl/libpointmatcher/commit/a9b72eb323#diff-af3b638bc2a3e6c650974192a53c7291R190) and back then just shadowed by _this_ (#1), the problem with libnabo. I will therefore create a new libpointmatcher issue and close this one. We can still reopen if it turns out to a wrong theory.
